sdk/devcenter/Azure.ResourceManager.DevCenter/src/autorest.md

```diff
@@ -9,21 +9,53 @@ generate-model-factory: false
 csharp: true
 library-name: DevCenter
 namespace: Azure.ResourceManager.DevCenter
-# default tag is a preview version
-require: https://github.com/Azure/azure-rest-api-specs/blob/1b3b9c1dd4d2c875997ea0b392dc71418fb1f28d/specification/devcenter/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/07c55de803057861912799405580ea9d022853fc/specification/devcenter/resource-manager/readme.md
```

Comment:
Set the generate-model-factory: false to generate-model-factory: true which enables the model factory.

Reply:
Updated